src/pybind/mgr/dashboard/frontend/src/app/shared/shared.module.ts

```diff
@@ -23,7 +24,8 @@ import { ServicesModule } from './services/services.module';
   ],
   declarations: [
     PasswordButtonDirective,
-    DimlessBinaryDirective
+    DimlessBinaryDirective,
+    AutofocusDirective
   ],
   exports: [
     ComponentsModule,
```

Comment:
`AutofocusDirective` should also be added to the `exports` to guarantee that it can be used by other modules.